### PR TITLE
7450: remove event listener

### PR DESCRIPTION
**Pull request by kkoskelin**
a listener was preventing `touchmove` scrolling on ipad (possibly others).  Original intent was to prevent the background from scrolling, but this is currently deemed a harmless price to pay so that these "tall" modals can scroll as desired for ipad users.
future strategy: prefer non-modal interfaces when content has any chance of over-running the height of the viewport.

### CODE REVIEW

**Review by matthopson — APPROVED** [review]

File: web-client/src/providers/socket.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const socketProvider = ({ socketRouter }) => {
           socket = createWebSocketClient(token);
           socket.onmessage = socketRouter(app);
           socket.onerror = error => {
-            applicationContext.logger.error('Websocket error detected', error);
+            console.error(error);
```
Comment (rachelschneiderman) [inline web-client/src/providers/socket.js:31]: Were these changes intentional?

Comment (kkoskelin) [reply web-client/src/providers/socket.js:31]: These are odd merge artifacts from another PR.
